packages/gafl-webapp-service/src/handlers/__tests__/error-handler.spec.js

```diff
@@ -88,6 +91,31 @@ describe('error-handler', () => {
         }
       })
     })
+
+    describe.each([[NEW_TRANSACTION.uri], [CONTROLLER.uri], [AGREED.uri]])('addLanguageCodeToUri', urlToCheck => {
+      it('called with expected arguments', async () => {
+        const request = getMockRequest()
+        const mockToolkit = getMockToolkit()
+        await errorHandler(request, mockToolkit)
+        expect(addLanguageCodeToUri).toHaveBeenCalledWith(request, urlToCheck)
+      })
+
+      // it.only('uri object returns correct', async () => {
+      //   const decoratedUri = Symbol('uri')
+      //   addLanguageCodeToUri.mockReturnValueOnce(decoratedUri)
+      //   const request = getMockRequest()
+      //   const mockToolkit = getMockToolkit()
+      //   await errorHandler(request, mockToolkit)
+      //   expect(mockToolkit.view).toHaveBeenCalledWith(
+      //     expect.any(String),
+      //     expect.objectContaining({
+      //       uri: expect.objectContaining({
+      //         decoratedUri
+      //       })
+      //     })
+      //   )
+      // })
```

Comment:
Thanks, this addresses my previous comment, although obviously shouldn't be commented or using `it.only`